.pylintrc

```diff
@@ -28,7 +28,7 @@ ignored-classes=numpy,scipy,autograd,toml,appdir,autograd,autograd.numpy,autogra
 # can either give multiple identifier separated by comma (,) or put this option
 # multiple time (only on the command line, not in the configuration file where
 # it should appear only once).
-disable=line-too-long,invalid-name,too-many-lines,redefined-builtin,too-many-locals,duplicate-code
+disable=line-too-long,invalid-name,too-many-lines,redefined-builtin,too-many-locals,duplicate-code,cyclic-import
```

Comment:
:+1: 

Maybe worth adding a note here, that the decision to disable cyclic import warnings was made within the team (instead of e.g., being something recommended/best practice, etc.).

Reply:
Yes I created a ticket to come back to this at a later point :)